ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -145,18 +151,12 @@ public void onEvent(ByteBufferAsyncWritableChannel.EventType e) {
    */
   void submitPutBlobOperation(BlobProperties blobProperties, byte[] userMetaData, ReadableStreamChannel channel,
       FutureResult<String> futureResult, Callback<String> callback) {
-    try {
-      PutOperation putOperation =
-          new PutOperation(routerConfig, routerMetrics, clusterMap, responseHandler, notificationSystem, userMetaData,
-              channel, futureResult, callback, routerCallback, chunkArrivalListener, kms, cryptoService,
-              cryptoJobHandler, time, blobProperties);
-      putOperations.add(putOperation);
-      putOperation.startReadingFromChannel();
-    } catch (RouterException e) {
-      routerMetrics.operationDequeuingRate.mark();
-      routerMetrics.onPutBlobError(e, blobProperties != null && blobProperties.isEncrypted());
-      NonBlockingRouter.completeOperation(futureResult, callback, null, e);
-    }
+    PutOperation putOperation =
```

Comment:
why was this try catch removed?

Reply:
The constructor no longer throws `RouterException`